kubernetes/cmsweb/hpa/crabserver-hpa.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: crabserver-hpa
   namespace: crab
 spec:
-  maxReplicas: 8
-  minReplicas: 5
+  maxReplicas: 10
+  minReplicas: 10
```

Comment:
@novicecpp 
minimum and maximum replicas should not be same. Otherwise there is no point of having hpa. Minimum replicaas should be less than 10

Reply:
The new minimum number of pods will be 10 and we do not want to use hpa. I change this value in case someone accidentally applies it.